packaging/linux/post_install.sh

```diff
@@ -13,24 +13,13 @@ set -u
 
 rootmount="/keybase"
 krbin="/usr/bin/keybase-redirector"
-rootConfigFile="/etc/keybase/config.json"
-disableConfigKey="disable-root-redirector"
-disableAutorestartConfigKey="disable-autorestart"
 
 redirector_enabled() {
-  disableRedirector="false"
-  if [ -r "$rootConfigFile" ] ; then
-    if keybase --standalone -c "$rootConfigFile" config get -d "$disableConfigKey" &> /dev/null ; then
-      disableRedirector="$(keybase --standalone -c "$rootConfigFile" config get -d "$disableConfigKey" 2> /dev/null)"
-    fi
-  fi
-  [ "$disableRedirector" != "true" ]
+  keybase --use-root-config-file config get --direct --assert-false --assert-ok-on-nil disable-root-redirector &> /dev/null
```

Comment:
Don't need `--standalone` on these?

Reply:
I added ```            if c.Bool("direct") {
                cl.SetForkCmd(libcmdline.NoFork)
                cl.SetLogForward(libcmdline.LogForwardNone)
            }   ```
which should basically "do" standalone automatically. Similar thing happens with `keybase base62`